internal/state/store.go

```diff
@@ -499,8 +558,7 @@ func (store dbStore) LoadValidators(height int64) (*types.ValidatorSet, error) {
 				fmt.Errorf("couldn't find validators at height %d (height %d was originally requested): %w",
 					lastStoredHeight,
 					height,
-					err,
-				)
+					err)
```

Comment:
I'm not sure which way is the convention in tendermint, but I typically leave the trailing paren on its own line for quickly identifiable symmetry with the opening. I think the standalone paren is typically more common in Go.